var/ramble/repos/builtin/applications/quantum-espresso/application.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2022-2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+import os
+from ramble.appkit import *
+from ramble.expander import Expander
+
+
+class QuantumEspresso(SpackApplication):
+    '''Define Quantum-Espresso application.
+    https://www.quantum-espresso.org/
```

Comment:
nit: Is this needed? Since it relies on the spack package which supports `url` is seems a bit surplus?